docs/notes/changelog.rst

```diff
@@ -6,6 +6,8 @@ Next Version
 
 **Breaking Changes**
 
+* Drop support of Python 3.5.
```

Comment:
An explanation why we're doing this would be good:

`* Drop support of Python 3.5 because a required dependency (llvmlite) does not support it anymore.`